### DISCUSSION TITLE
Wrong recordings folders timestamp? Is it a bug?

### DISCUSSION THREAD

**Discussion by TarcisioMenezes** [blakeblackshear/frigate#6165] [Ask A Question] [answered]
Hello guys! I noticed that the folders inside `recordings` have been created with a wrong timestamp. 
For example:
![image](https://user-images.githubusercontent.com/12872368/233167864-27b6cc53-4884-4897-af05-c2e5d4fe9be1.png)
 This was recorded on 2023-04-18 > at 11pm.

I'm in Central Daylight Time (GMT-5)
unRAID 6.11.5 / Frigate 0.12.0

Host seems to have the right time and date config.

![image](https://user-images.githubusercontent.com/12872368/233168440-8caddff6-8a6e-4fd7-9ebe-71be7c63aba1.png)

Also, 
`-e TZ="America/Chicago"` 
`-v '/etc/localtime':'/etc/localtime':'rw'` 
are already included by default in the docker run command.

Does anyone know how to change that? Seems Frigate is using GMT-0 to label the folders.
Frigate UI shows the right time for logs, events, recordings and etc. 
Is it a bug?

**Comment by NickM-27** [comment, marked as answer]
the folders are stored in UTC, this is explained in the 0.12 breaking changes and has a dedicated release section. This is by design and won't be changing (previous versions were not stored in UTC, just fyi)

**Reply by TarcisioMenezes** [reply]
I missed that part in the docs.
Thanks for the fast reply, Nick!

**Reply by ljmerza** [reply]
why? this is bad ux if youre trying to find a video.

**Reply by NickM-27** [reply]
because not using UTC causes issues with daylight savings time, makes it more difficult to support timezone config option, among other problems. 

Losing recording footage due to daylight savings time is bad UX. You should not be using the file structure to find recordings, you should just use the webUI
